nanome/_internal/_plugin_instance.py

```diff
@@ -3,7 +3,11 @@
 from nanome._internal._network._commands._callbacks import _Messages
```

Comment:
Test with MD

Reply:
MD became super slow, but this problem is already on master or 0.0.3